src/stack/Stack.java

```diff
@@ -19,38 +21,73 @@ public Stack(int size) {
 	 * Add element into stack
 	 */
 	public void push(int value) {
-
+		if (tos == data.length-1) {
+			ensureCapacity();
+			data[++tos] = value;
+		} else {
+			data[++tos] = value;
+		}
 	}
 
 	/**
 	 * Get element from the stack
 	 * @return
 	 */
 	public int pop() {
-
-		return 0;
+		if (tos == -1) {
+			System.out.println("Stack is empty");
+			return 0;
+		} else {
+			return data[tos--];
```

Comment:
If you use **return** statement inside **ELSE**, so you don't need **ELSE** statement. Just put **return data[tos--];** after **IF statement**